packages/components/CHANGELOG.md

```diff
@@ -56,6 +56,7 @@
 -   `Autocomplete`: use Popover's new `placement` prop instead of legacy `position` prop ([#44396](https://github.com/WordPress/gutenberg/pull/44396/)).
 -   `SelectControl`: Add `onChange`, `onBlur` and `onFocus` to storybook actions ([#45432](https://github.com/WordPress/gutenberg/pull/45432/)).
 -   `FontSizePicker`: Add more comprehensive tests ([#45298](https://github.com/WordPress/gutenberg/pull/45298)).
+-   `Snackbar`: Convert to TypeScript ([#45472](https://github.com/WordPress/gutenberg/pull/45472)).
```

Comment:
Also, we should rebase this PR and move this CHANGELOG entry under "Unreleased", in the "### Internal" section

Reply:
Ah, good idea. I just did that.

Is it alright that I didn't do `git rebase`, I just merged in `trunk` and resolved the conflicts?

Reply:
Sure, you can either merge or rebase — your preference.